x-pack/plugin/sql/qa/src/main/resources/null.sql-spec

```diff
@@ -0,0 +1,9 @@
+//
+// Null expressions
+//
+
+coalesceField
+SELECT COALESCE(null, ABS(emp_no) + 1) AS c FROM test_emp ORDER BY emp_no LIMIT 5;
+
+coalesceHaving
+SELECT COALESCE(languages, 0) c FROM test_emp GROUP BY languages ORDER BY languages;
```

Comment:
there is no having condition here. can we add one?

Reply:
Also, can you add a test when COALESCE is used in the WHERE filter (no aggregations).

Reply:
done.